documentation/book/con-requests-kafka-bridge.adoc

```diff
@@ -5,21 +5,31 @@
 [id='con-requests-kafka-bridge-{context}']
 = Requests to the Kafka Bridge
 
-Specify data formats and HTTP headers to ensure valid requests are submitted to the Kafka Bridge. 
+Specify data formats and HTTP headers to ensure valid requests are submitted to the Kafka Bridge.
+
+API request and response bodies are always encoded as JSON.
 
 == Content Type headers
 
-API request and response bodies are always encoded as JSON.
+A `Content-Type` header must be submitted for all requests (apart from the exception described below). 
+
+Consumer operations (`/consumers` endpoints) and producer operations (`/topics` endpoints) require different `Content-Type` headers.  
+
+.Content-Type headers for consumer operations
+
+Regardless of the link:#embedded-data-format[embedded data format], `POST` requests for consumer operations must provide the following `Content-Type` header if the request body contains data: 
 
-* When performing consumer operations, `POST` requests must provide the following `Content-Type` header if there is a non-empty body:
-+
 [source,http,subs=+quotes]
 ----
 Content-Type: application/vnd.kafka.v2+json
 ----
 
-* When performing producer operations, `POST` requests must provide `Content-Type` headers specifying the desired _embedded data format_, either `json` or `binary`, as shown in the following table.
-+
+Do not send a `Content-Type` header if the `POST` request body is empty, or the request will fail.
```

Comment:
maybe we could explain why it will fail? It's related to the OpenAPI validation.

Reply:
Alright. I've rephrased this tricky sentence for the third time! I hope it's clear.